bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -139,7 +139,7 @@ public final class Image extends Resource implements Drawable {
 	/**
 	 * Attribute to cache current device zoom level
 	 */
-	private int currentDeviceZoom = 100;
+	int currentDeviceZoom = 100;
```

Comment:
Why? Is this necessary for follow-up PRs? If it is, keep in mind that this allows writing into it so maybe you better provide a getter insted

```suggestion
	private int currentDeviceZoom = 100;
```

Reply:
Introducing a public getter/setter will change the API and it is a Win32 specific implementation so we should not do that.